app/_locales/en/messages.json

```diff
@@ -4679,9 +4679,39 @@
   "simulationsSettingSubHeader": {
     "message": "Estimate balance changes"
   },
+  "siweAccount": {
+    "message": "Account"
+  },
+  "siweChainID": {
+    "message": "Chain ID"
+  },
+  "siweIssued": {
+    "message": "Issued"
+  },
+  "siweNetwork": {
+    "message": "Network"
+  },
+  "siweNonce": {
+    "message": "Nonce"
+  },
+  "siweOriginalMessage": {
+    "message": "Message"
+  },
```

Comment:
for generic translations like this, I think it'd be better to use direct key to message translations. e.g. there is one that exists for Message
```
  "message": {
    "message": "Message"
  },
```
similarly for "Account", "Chain ID", "Nonce", and "Version"

I think the others could be direct translations as well. "Request Id" and "Resources" are questionable and could be left as is.

I believe using direct translations can be better because:
- It'd be easier to grasp the context through the files where they were called. e.g. siweOriginalMessage leaves room for interpretation whereas message can indicate a direct translation
- Less noise and maintenance in the translation files

Reply:
I did the change